pkg/util/mount/mount_windows.go

```diff
@@ -74,8 +74,23 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 			return nil
 		}
 
-		// empty implementation for mounting azure file
-		return os.MkdirAll(target, 0755)
+		cmdLine := fmt.Sprintf(`$User = "%s";$PWord = ConvertTo-SecureString -String "%s" -AsPlainText -Force;`+
```

Comment:
please leave a note here explaining only smb mount is expected here and log a warning otherwise.

Reply:
done